stl/inc/complex

```diff
@@ -1877,7 +1877,7 @@ _Ty _Fabs(const complex<_Ty>& _Left, int* _Pexp) { // Used by sqrt(), return mag
             _Bv    = _Bv * static_cast<_Ty>(0.0625);
         } else {
             constexpr _Ty _Flt_eps = _Ctraits<_Ty>::_Flt_eps();
-            // TRANSITION, workaround for non floating point _Ty
+            // TRANSITION, workaround for non floating-point _Ty

```

Comment:
"non" isn't a word; this should be "non-floating-point". (At least one more occurrence - please grep.)

Reply:
Fixed all of them except for a quoted compiler warning message.